### PR TITLE
Bug 1955478: drop high-cardinality metrics from kube-state-metrics which aren't used

### PR DESCRIPTION
**Pull request by simonpasquier**
* [ ] I added CHANGELOG entry for this change.
* [X] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: jsonnet/main.jsonnet [hunk 1]
```diff
@@ -321,6 +321,7 @@ local inCluster =
     openshiftStateMetrics: openshiftStateMetrics($.values.openshiftStateMetrics),
   } +
   (import './anti-affinity.libsonnet') +
+  (import 'github.com/prometheus-operator/kube-prometheus/jsonnet/kube-prometheus/addons/ksm-lite.libsonnet') +
```
Comment (simonpasquier) [inline jsonnet/main.jsonnet:324]: @paulfantom please confirm it is the right place to import the addon :)

Comment (paulfantom) [reply jsonnet/main.jsonnet:324]: It is :+1: 

/lgtm
/hold cancel

**Pull request merged**